kickstart/provision.erb

```diff
@@ -183,12 +183,14 @@ echo "updating system time"
 echo 'proxy = <%= proxy_uri %>' >> /etc/yum.conf
 <% end -%>
 
+<% unless @host.param_false?('update_base_packages') -%>
```

Comment:
@domcleal I couldn't see what `package_upgrade` actually does from community_templates, I'm guessing it's something to do with `yum **update**` vs `yum **upgrade**`

What would you suggest as an acceptable parameter name to define whether or not `yum update` should happen or not ?

Sure I'll add it to the readme once we've settled on a parameter name.